cfbs/commands.py

```diff
@@ -1072,3 +1072,32 @@ def _compare_list(a, b):
     write_json(dst_path, data)
 
     return 0
+
+
+def get_input_command(name, dst_path):
+    config = CFBSConfig.get_instance()
+    module = config.get_module_from_build(name)
+    if module is None:
+        log.error("Module '%s' not found" % name)
+        return 1
+
+    if "input" not in module:
+        log.error("Module '%s' does not accept input" % name)
+        return 1
+
+    src_path = os.path.join(name, "input.json")
+    data = read_json(src_path)
+    if data is None:
+        log.debug("Loaded input from module '%s' definition" % name)
+        data = module["input"]
+    else:
+        log.debug("Loaded input from '%s'" % src_path)
+
+    if dst_path is None:
+        log.debug("Writing json to stdout")
+        print(pretty(data))
+    else:
+        log.debug("Writing json to '%s'" % dst_path)
+        write_json(dst_path, data)
```

Comment:
I would expect `get-input module some-file` to extract the input from `some-file`. Again, I'd prefer an extra option for the output.